src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -9,32 +9,18 @@
 using System.Security.Cryptography.X509Certificates;
 using System.Threading;
 using System.Threading.Tasks;
-#if TARGET_BROWSER
-using HttpHandlerType = System.Net.Http.BrowserHttpHandler;
-#else
 using HttpHandlerType = System.Net.Http.SocketsHttpHandler;
```

Comment:
Do we still need the alias if the file is only for browsers?

Reply:
This file is share file